src/infra/instrumentation/instrumentation-agent.js

```diff
@@ -42,7 +42,7 @@ class InstrumentationAgent {
     if (process.env.DATADOG_API_KEY) {
       this.metrics = metrics;
       metrics.init({
-        host: process.env.HOST,
+        host: process.env.DATADOG_HOST || process.env.HOST,
```

Comment:
Actually I would fallback to a hardcoded value here. It will be easier to manage and make sure we don't send a different host by default for each new connector if the `DATADOG_HOST` variable is not set.

Reply:
Hm, not sure about putting our infra specific URL in this library. What we can do differently to achieve what's important to you is to exit the process early with a descriptive error when `DATADOG_HOST` env var is missing and we have `DATADOG_API_KEY` present.
@sbellity @gabrielnau?